src/app/multi-upload/multi-upload.component.html

```diff
@@ -0,0 +1,13 @@
+<div>
+  <input id="multi-file-upload" type="file" [accept]="acceptedFiles" (change)="click($event)" multiple>
+  <label for="multi-file-upload" class="dragzone"
+         (drag)="preventDefault($event)"
+         (dragover)="preventDefault($event)"
+         (dragenter)="preventDefault($event)"
+         (drop)="drop($event)">
```

Comment:
It's not working, when I only prevent on `dragover` either `dragenter`, it's only working with all 4 (drop calls preventDefault)